build.sc

```diff
@@ -230,6 +237,7 @@ trait BenchModule extends CommonModule{
     ivy"io.argonaut::argonaut:6.2",
     ivy"org.json4s::json4s-ast:3.5.2",
     ivy"com.lihaoyi::sourcecode:0.1.4",
+    ivy"com.avsystem.commons::commons-core:1.26.3",
```

Comment:
@ghik can you try replacing this line with
```scala
ivy"com.avsystem.commons::commons-core::1.26.3"
```

(Note the double `::` on the right hand side of `commons-core`)

And see if that lets you run the JS benchmarks?

Reply:
That was it, thanks!